src/main/java/Duke.java

```diff
@@ -1,10 +1,197 @@
+                }
+
+                break;
+
+            case "event":
+                try{
+                    addEvent(taskLists, userInputSplit);
+                }
+                catch(StringIndexOutOfBoundsException e){
+                    System.out.println(e.toString());
+                };
+                break;
+
+            case "deadline":
+                addDeadline(taskLists, userInputSplit);
+                break;
+
+
+            case "unmark":
+                int markNotDoneIndex = Integer.parseInt(userInputSplit[1]) - 1;
+                taskLists.get(markNotDoneIndex).setIsDone(false);
+                printSetNotDoneMessage(taskLists.get(markNotDoneIndex));
+                break;
+
+            case "delete":
+                int deleteIndex = Integer.parseInt(userInputSplit[1]) - 1;
+                deleteTask(taskLists, deleteIndex);
+                break;
+
+                default:
+                printDontKnowMessage();
+            }
+            userInput = getUserInput();
+        }
+
+        printByeMessage();
     }
+
+    private static void deleteTask(ArrayList<Task> taskLists, int deleteIndex) {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Noted. I've removed this task:");
+        System.out.println(String.format("       %s", taskLists.get(deleteIndex).toString()));
+        taskLists.remove(deleteIndex);
+        Task.numOfTasks --;
+        System.out.println("     Now you have " + Task.numOfTasks + " tasks in the list.");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    private static void addTodo(ArrayList<Task> taskLists, String[] userInputSplit) throws EmptyDescriptionException {
+        if (userInputSplit.length < 2){
+            throw new EmptyDescriptionException();
+        }
+        String[] inputArrayWithoutType = Arrays.copyOfRange(userInputSplit,1, userInputSplit.length);
+        String description = String.join(" ", inputArrayWithoutType);
+        ToDo newToDo = new ToDo(description);
+        taskLists.add(newToDo);
+        Task.numOfTasks ++;
+        printEchoInput(newToDo);
+    }
+
+    private static void addEvent(ArrayList<Task> taskLists, String[] userInputSplit) throws StringIndexOutOfBoundsException{
+
+        String[] inputArrayWithoutType = Arrays.copyOfRange(userInputSplit, 1, userInputSplit.length);
+        String inputWithoutType = String.join(" ", inputArrayWithoutType);
+        if (inputWithoutType.length() < 3){
+            throw new StringIndexOutOfBoundsException();
+        }
+        String description = inputWithoutType.split(" /at ")[0];
+        String time = inputWithoutType.split(" /at ")[1];
+        Event newEvent = new Event(description, time);
+        taskLists.add(newEvent);
+        Task.numOfTasks ++;
+        printEchoInput(newEvent);
+    }
+    // add exception
```

Comment:
This seems like a reminder to yourself to add exceptions. Avoid personal comments that are private notes to yourself, instead comments should be written to the reader to improve readability. If not necessary here, can remove it altogether.

Code quality guideline: Write to the Reader